database/entity/0060-update_communities_table/Community.ts

```diff
@@ -25,13 +25,13 @@ export class Community extends BaseEntity {
   endPoint: string
 
   @Column({ name: 'last_announced_at', type: 'datetime', nullable: true })
-  lastAnnouncedAt: Date
+  lastAnnouncedAt: Date | null
```

Comment:
Not certain if ``| null`` could be replaced with 
```suggestion
  lastAnnouncedAt?: Date
```

Reply:
Perhaps, i'am not sure too.
It is copied from other entities where the date is optional too